core/src/main/resources/hudson/model/Messages.properties

```
@@ -129,6 +129,7 @@ Hudson.Computer.Caption=Built-In Node
 Hudson.Computer.DisplayName=Built-In Node
 Hudson.ControlCodeNotAllowed=No control code is allowed: {0}
 Hudson.DisplayName=Jenkins
+Hudson.JdkDisplayName=JDK
```

maybe
```suggestion
JDK.displayName=JDK
```
since this is not about `Hudson.java` (there is no strict rule)

---

Agree, renamed property key.